### PR TITLE
Smart contract auto generated docs

### LINKED ISSUE #49: Blockchain components/Smart Contracts
Each mainnet and sidechain smart contract need a description of its functions and data structures.

### PR DESCRIPTION
**Pull request by alexvanin**
Closes #49 

## Issues

At first I wanted to have `<xxx> contract` header to be at level 2. However I wasn't able to make contract name starting with capital letter in template, so it either should be at level 3 or be `Contract <xxx>`. I decided to go with first option.

Because of that public methods placed in level 5 headers and that may look odd in the output pdf.

## Todo

- [x] Alphabet
- [x] Audit
- [x] Balance
- [x] Container
- [x] NeoFS
- [x] NeoFSID
- [x] Netmap
- [x] Processing
- [x] Proxy
- [x] Reputation

### CODE REVIEW

**Comment by realloc** [comment]
Escaped dots and commas look kinda weird. Is there any easy fix for that?

**Comment by alexvanin** [comment]
> Escaped dots and commas look kinda weird. Is there any easy fix for that?

I tried `-f plain` format but it didn't work out. Even gomarkdoc's [README.md](https://raw.githubusercontent.com/princjef/gomarkdoc/master/README.md) contains escaped symbols. However it is rendered just fine in the web.

File: 06-blockchain/02-contracts.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## NeoFS Smart Contracts
+
+This section contains brief description about the role of contracts and
+autogenerated documentation from source code.
```
Comment (realloc) [inline 06-blockchain/02-contracts.md:4]: The note about auto-generated nature of the following chapter doesn't add any good, let's omit it.

**Review by realloc — DISMISSED** [review]
The resulting PDF looks good.

**Review by realloc — APPROVED** [review]

**Pull request merged**